packages/store-ui/src/atoms/Slider/stories/Slider.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import type { Meta, Story } from '@storybook/react'
+import React from 'react'
+
+import type { ComponentArgTypes } from '../../../typings/utils'
+import type { SliderProps } from '../Slider'
+import Component from '../Slider'
+import mdx from './Slider.mdx'
+
+const SliderTemplate: Story<SliderProps> = (props) => <Component {...props} />
+
+export const Slider = SliderTemplate.bind({})
+
+const argTypes: ComponentArgTypes<SliderProps> = {
+  min: {
+    control: { type: 'text' },
+    defaultValue: '0',
+  },
+  max: {
+    control: { type: 'text' },
```

Comment:
```suggestion
    control: { type: 'number' },
```

Reply:
If you want to add min and max for these values it would look like this (same as Bullets.stories.tsx):

```
control: { type: 'number', min: 0 },
```

Reply:
Hmm, nice.